### PR TITLE
Add codeclimate/eslint integration

### PR DESCRIPTION
**Pull request by ghost**
Add codeclimate/eslint integration

### CODE REVIEW

**Comment by devs-deep** [comment]
:white_check_mark: coverage increased from 0% to 74.83%

File: src/deep-helloworld/frontend/bootstrap.js [hunk 1]
```diff
@@ -11,7 +11,9 @@
 
     xhttp.onreadystatechange = function () {
       if (xhttp.readyState === 4 && xhttp.status === 200) {
-        callback(xhttp.responseText);
+        callback(xhttp.responseText)
+
+        return;
```
Comment (vcernomschi) [inline src/deep-helloworld/frontend/bootstrap.js]: Please update corresponding to latest skeleton and remove `return` statement.

File: src/deep-helloworld/hook.post-deploy.js [hunk 2]
```diff
@@ -30,5 +32,7 @@ var exports = module.exports = function(callback) {
     }
 
     callback();
-  }.bind(this));
-};
\ No newline at end of file
+
+    return;
```
Comment (vcernomschi) [inline src/deep-helloworld/hook.post-deploy.js]: Please update corresponding to latest skeleton and remove `return` statement.

File: src/deep-helloworld/frontend/bootstrap.js [hunk 3]
```diff
@@ -41,4 +43,4 @@
       });
     });
   });
-})(DeepFramework);
+}(DeepFramework));
```
Comment (vcernomschi) [inline src/deep-helloworld/frontend/bootstrap.js]: Please undo to inner wrap, there are no sense to change. For more details please refer to: http://eslint.org/docs/rules/wrap-iife

**Comment by devs-deep** [comment]
:white_check_mark: coverage increased from 0% to 75.33%

**Pull request merged**